v2/internal/frontend/desktop/darwin/WailsContext.m

```diff
@@ -258,25 +258,28 @@ - (void) CreateWindow:(int)width :(int)height :(bool)frameless :(bool)resizable
                   forMainFrameOnly:false];
         [userContentController addUserScript:initScript];
     }
-    
-    self.webview = [WKWebView alloc];
+
+    self.webview = [WailsWebView alloc];
```

Comment:
The important change is here. We use our extended implementation of WKWebkit here